.github/workflows/lint.yml

```diff
@@ -68,11 +68,15 @@ jobs:
         run: |
           echo "::add-matcher::.github/workflows/matchers/pylint.json"
           tox -e lint
+          echo "::remove-matcher owner=pylint-error::"
```

Comment:
I'm not sure if we have to remove them across separate steps. Maybe someone could confirm.

Reply:
I think removing the matcher is safest. It can avoid the previous matcher also matching the errors.

Reply:
Ack. I am not sure the state of matchers survive across steps; but I guess it won't make it worse.